test/_windows.py

```diff
@@ -556,6 +562,19 @@ def test_cwd_64(self):
         p = psutil.Process(self.proc64.pid)
         self.assertEqual(p.cwd(), os.getcwd())
 
+    def test_environ_32(self):
+        p = psutil.Process(self.proc32.pid)
+        e = p.environ()
+        self.assertIn("THINK_OF_A_NUMBER", e)
+        self.assertEquals(e["THINK_OF_A_NUMBER"], str(os.getpid()))
+
+    def test_environ_64(self):
+        p = psutil.Process(self.proc64.pid)
+        time.sleep(0.1)
```

Comment:
Hmm... this is interesting. If that's the case then maybe we should check that condition (ERROR_PARTIAL_READ) and raise NoSuchProcess if pid does not exist anymore. Makes sense?